test/conformance/source/environment.cpp

```diff
@@ -266,6 +266,17 @@ std::string KernelsEnvironment::getSupportedILPostfix(uint32_t device_index) {
         return {};
     }
 
+    // special case for AMD as it doesn't support IL.
```

Comment:
Is this a TODO? should be there an issue to fix IL then?

Reply:
Possibly, although from speaking to the team implementing the plugin they are likely to not support this query as the HIP does not accept 'IL' as such.